pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -374,7 +374,7 @@ def run_evaluation(self, test_mode: bool = False):
 
         # cap max batches to 1 when using fast_dev_run
         if self.fast_dev_run:
-            max_batches = [1]
+            max_batches = [1] * len(dataloaders)
```

Comment:
Not sure how to make assertions in test to check num_batches since max_batches is a local variable and no other member variable knows about the num_batches when we use `fast_dev_run=True`.

Reply:
I suggest making `self.limit_{mode}_batches` to 1 a few lines above in case of `fast_dev_run=True` so that `num_{mode}_batches` will be made as such and we don't have to explicitly check it again and change `max_batches` later. Hopefully, this won't break anything.